tools/build_release_notes.py

```diff
@@ -144,8 +144,7 @@ def push_website_head_commit(
 def run(CONF):
     versions_repo = setup_versions_repository(CONF)
 
-    packages_names = (CONF.get('common').get('packages')
-                      or packages_manager.discover_packages())
+    packages_names = packages_manager.discover_packages()
```

Comment:
I don't know, just thought we would like to keep consistent with build-packages. Ok then.